### DISCUSSION TITLE
Welcome to deprem-yardim-projesi Discussions!

### DISCUSSION THREAD

**Discussion by ramazansancar** [acikkaynak/deprem-yardim-projesi#25] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ozcamseyma** [comment]
Afet İçin Toplanılan Yardımların Takibi Projesi:
Yardım gönderdikten sonra nerede olduğunu ve hangi yerde dağıtıldığını görebildiğimiz bir proje.
Bu yardımlar gündemimizde de olduğu gibi yerlere atılarak heba edilmiş durumda. 
Örneğin, Antakya'ya gidiyorsa bu yardım bunun takibi yapılmadır ve güvenli bir şekilde dağıtımı yapılıyor mu orada bu da çok önemli yağmalama olaylarını engellemek için. İnsanlar bu tip durumlar olunca verilen yardımın zarar görmeden teslim edilmesi konusunda tedirgin olabiliyor ve bu çoğu kişide var tespitlerimi göz önünde bulundurduğum zaman.
Ve her şehirden oraya vardığının teyidini almalıyız oradaki sorumlu gönüllüler aracılığı ile.
Yardımı gönderen kişiye de iki teslimat bilgisi gitmiş olucak.(teslim edildi ve teslim alındı şeklinde)
Projede temel amaçlar, yardım isteyen kişi ile irtibata geçmeden tamamen gönüllü olarak gönderilen yardımların iletilmesi. Bu yardımı alan kişi kimden geldiğini bilmemeli çünkü bu durum yardıma ihtiyacı olan kişi için çok hassas bir durum(çekinme ya da acizlik gibi duygular hissettirebilir) Tamamen gönüllük esaslı olan yardımların takibi olmalıdır.

**Comment by Arikitokurin** [comment]
Buraya yazmak zorunda kaldım çünkü başlık açtığımda kaydetmeden başa atiyor herhangi bir onay süreci yoksa. 

Çocuk kayıpları fazla. Bu gündeme gelmişti aslında ve legalden dönmüştü diye anımsıyorum. Ancak yine de belirtmek istedim.

**Reply by sercangoger** [reply]
fikir olarak açıyorum hocam hemen sizin yerinize.
